google-mysql.yml

```diff
@@ -141,6 +141,31 @@ provision:
     constraints:
       minimum: 0
       maximum: 7
+  - field_name: highly_available
+    type: boolean
+    details: |
+      Enable regional high availability for the service instance. Requires transaction log backups to be enabled. See
+      https://cloud.google.com/sql/docs/mysql/high-availability for more details.
```

Comment:
That's a good catch, @zucchinidev!
I think it would be a better idea to expand the `backups_enabled` property to take log backups into account.
Given that `backups_retain_number` is positive by default, they have to be disabled explicitly.

Reply:
Oh, great. Would you change it in this PR? or do you want to create another one?

Reply:
It will be this one.

Reply:
Fun fact. When `backups_enabled` is set to true, and `backups_retain_number` is set to `0`, the actual value for `backups_retain_number` is actually `7`. Apparently, this happens if you enable backups and don't specify the retention count.